modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -55,10 +55,16 @@ If a watchdog device is unavailable, the `SelfNodeRemediationConfig` CR uses a s
 [id="understanding-self-node-remediation-remediation-template-config_{context}"]
 == Understanding the Self Node Remediation Template configuration
 
-The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` CR with the name `self-node-remediation-default-template` in the Self Node Remediation Operator's namespace. This CR defines the remediation strategy for the nodes.
+The Self Node Remediation Operator also creates the `SelfNodeRemediationTemplate` Custom Resource Definition (CRD). This CRD defines the remediation strategy for the nodes. The following remediation strategies are available:
 
-The default remediation strategy is `NodeDeletion` that removes the `node` object.
-In {product-title} 4.10, the Self Node Remediation Operator introduced a new remediation strategy called `ResourceDeletion`. The `ResourceDeletion` remediation strategy removes the pods and associated volume attachments on the node rather than the `node` object. This strategy helps to recover workloads faster.
+ResourceDeletion:: This remediation strategy removes the pods and associated volume attachments on the node rather than the node object. This strategy helps to recover workloads faster. `ResourceDeletion` is the default remediation strategy.
+
+NodeDeletion:: This remediation strategy removes the node object.
```

Comment:
Should "NodeDeletion" be in back-ticks? I'm not entirely sure, given that this is a list heading. But, I thought I'd mention it here for your review.